### PR TITLE
Remved take(1) from getTranslation() to allow updating the stream mor…

### PR DESCRIPTION
**Pull request by kkotak**
Fix for - https://github.com/ngx-translate/core/issues/736

### CODE REVIEW

File: lib/src/translate.service.ts [hunk 1]
```diff
@@ -264,7 +264,7 @@ export class TranslateService {
         this.pending = true;
         this.loadingTranslations = this.currentLoader.getTranslation(lang).pipe(share());
 
-        this.loadingTranslations.pipe(take(1))
+        this.loadingTranslations.pipe()
```
Comment (alexndreazevedo) [inline lib/src/translate.service.ts]: There is no need to the `pipe` if no operators are set.

**Comment by ocombe** [comment]
If you remove the `take(1)` you'll run into memory leaks, and you'll have multiple loading translations observables in memory, triggering multiple times because there is no unsubscribe.

**Comment by alexndreazevedo** [comment]
@ocombe, IMHO the loader should be the responsible for control when the translations should be loaded, not the core.

**Comment by ocombe** [comment]
Agreed, but it would be a super breaking change, not sure how we could do that change smoothly

**Comment by alexndreazevedo** [comment]
What about placing it as a setting disabled by default on `TranslateModuleConfig`? Additionally, the `ngx-translate/http-loader` could return an observable with `take(1)`.

**Comment by Nikki1993** [comment]
Hello there o/ any update on the matter? Recently hit a similar scenario where I wish to pre-cache translations via LocalStorage to reduce the loading times but hit the snag where translations are not updated (by design).

**Comment by MKhowaja** [comment]
With regards to this change being too breaking @ocombe I have a PR here with a different solution that should solve the same problem:
https://github.com/ngx-translate/core/pull/1028
Let me know what you guys think!

**Comment by ocombe** [comment]
Closing this as the other PR is merged

**Pull request closed without merging**